src/Sarif.UnitTests/Writers/SarifLoggerTests.cs

```diff
@@ -65,7 +65,7 @@ public void SarifLogger_RedactedCommandLine()
                         // the directory of the test driver (the location of which we can't retrieve
                         // from Assembly.GetEntryAssembly() as we are running in an AppDomain).
                         pathToExe = pathToExe.Substring(0, pathToExe.Length - @"\Extensions".Length);
-                        tokensToRedact = new string[] { "TestExecution", pathToExe };
```

Comment:
What bug is being fixed here? I just ran the SARIF SDK tests from VS and this one failed; is this the fix for that?